### DISCUSSION TITLE
how to disable user-agent restriction?

### DISCUSSION THREAD

**Discussion by nphariharan** [apache/apisix#10306] [Q&A]
Hi All,
For some of the routes, we want to disable user-agent header.
 For one of the route, I disabled the user-agent by applying the following rule.

    ua-restriction:
      bypass_missing: true

however it is not working? We need your support here.

Thanks,
Hariharan.

**Comment by kayx23** [comment]
It only bypasses when the `User-Agent` header is not present in the request, per the [doc](https://apisix.apache.org/docs/apisix/plugins/ua-restriction/) and here:

https://github.com/apache/apisix/blob/3a87611f93fb2de9fb36b3f87d3bc7a530ca5f52/apisix/plugins/ua-restriction.lua#L152-L161
